core-bundle/src/Resources/contao/themes/flexible/basic.css

```diff
@@ -298,13 +303,17 @@ input[type="search"]::-webkit-search-decoration {
 	.tl_text::placeholder,.tl_text_2::placeholder,.tl_text_3::placeholder,.tl_text_4::placeholder,.tl_textarea::placeholder,.tl_text_unit::placeholder,.tl_text_trbl::placeholder,.tl_text_interval::placeholder {
 		line-height:18px;
 	}
+	input[type="search"] {
+		padding-top:1px;
+		padding-bottom:1px;
+	}
```

Comment:
In case you are wondering why this needs to be adjusted on Firefox:

<img width="282" alt="" src="https://user-images.githubusercontent.com/1192057/85742516-97ee8e80-b703-11ea-88a8-ca7277971a43.png">

Firefox offsets the text inside the input and select fields, so I had to make it wrong on purpose to match the rest! 🤯